components/docs-chef-io/content/automate/applications_setup.md

```diff
@@ -248,11 +248,10 @@ If you are using a certificate signed by a trusted certificate authority instead
 For more information, check out this further explanation on how to [configure Builder to authenticate via Chef Automate](https://github.com/habitat-sh/on-prem-builder).
 ## Related Resources
 
-For more information, see the [Chef Habitat documentation](https://www.habitat.sh/docs).
+For more information, see the [Chef Habitat documentation]({{< relref "/habitat" >}}).
 In particular, see the entries on:
 
-* [Chef Habitat Best Practices](https://www.habitat.sh/docs/best-practices/)
```

Comment:
This one got dropped; was that intentional?

Reply:
That page doesn't exist anymore and now redirects to docs.chef.io/habitat, which is the same as the link on line 251.